internal/cli/atlas/deployments/delete.go

```diff
@@ -61,16 +118,52 @@ func (opts *DeleteOpts) Run(ctx context.Context) error {
 	})
 }
 
+func (opts *DeleteOpts) PostRun() error {
+	if !opts.EnableWatch || !opts.IsAtlasDeploymentType() {
+		return nil
+	}
+
+	watcher := watchers.NewWatcher(
+		*watchers.ClusterDeleted,
+		watchers.NewAtlasClusterStateDescriber(
+			opts.atlasStore.(store.AtlasClusterDescriber),
+			opts.ProjectID,
+			opts.Entry,
+		),
+	)
+
+	watcher.Timeout = time.Duration(opts.Timeout)
+	if err := opts.WatchWatcher(watcher); err != nil {
+		return err
+	}
+
+	return opts.Print(nil)
+}
+
 // atlas deployments delete <clusterName>.
 func DeleteBuilder() *cobra.Command {
 	opts := &DeleteOpts{
-		DeleteOpts: cli.NewDeleteOpts("Deployment '%s' deleted\n", "Deployment not deleted"),
+		DeleteOpts: cli.NewDeleteOpts(deleteSuccessMessage, deleteFailMessage),
 	}
 	cmd := &cobra.Command{
-		Use:     "delete [deploymentName]",
-		Short:   "Delete a deployment.",
+		Use:   "delete [deploymentName]",
+		Short: "Delete a deployment.",
+		Long: `The command prompts you to confirm the operation when you run the command without the --force option. 
+		
+Deleting an Atlas deployment also deletes any backup snapshots for that cluster.
+Deleting a Local deployment also deletes any local data volumes.
+
+` + fmt.Sprintf(usage.RequiredRole, "Project Owner"),
```

Comment:
we should probably remove this as it's only true for local

Reply:
this translates to: `To use this command, you must authenticate with a user account or an API key that has the Project Owner role.` which is still correct for Atlas deployments. Your suggestion is to remove it entirely or to adjust it to something like: `To use this command with Atlas deployments, you must authenticate with a user account or an API key that has the Project Owner role.`